packit_service/service/events.py

```diff
@@ -518,9 +552,10 @@ def pre_check(self):
 
     def get_dict(self, default_dict: Optional[Dict] = None) -> dict:
         d = self.__dict__
-        d.pop("build_pg")
+        build_pg = d.pop("build_pg")
         result = super().get_dict(d)
         result["topic"] = result["topic"].value
+        self.build_pg = build_pg
```

Comment:
Honestly, I am not 100% sure what I am doing here -- why we need to edit `__dict__` here just for the get method?
(I understand, that the `build_pg` is not serializable.)

Reply:
this is all just for sake of serialization and the way this is implemented atm

Reply:
Is this change safe? (I was having weird errors in tests because of the missing attribute `build_pg`...)

Reply:
should be fine